pimd/pim_cmd.c

```diff
@@ -5803,7 +5803,7 @@ static void show_mroute(struct pim_instance *pim, struct vty *vty,
 		vty_out(vty,
 			"       R - RP-bit set, F - Register flag, T - SPT-bit set\n");
 		vty_out(vty,
-			"\nSource          Group           Flags       Proto  Input            Output           TTL  Uptime\n");
+			"\nSource          Group           Flags   Proto  Input            Output           TTL  Uptime\n");
```

Comment:
curious: could you show how the output looks before and after?

Reply:
I had earlier introduced Flags which can take SCPRFT max, so deleting the extra space which is needed